sktime/base/_meta.py

```diff
@@ -713,6 +714,20 @@ def _tagchain_is_linked_set(
         else:
             self.set_tags(**{mid_tag_name: mid_tag_val_not})
 
+    def _sk_visual_block_(self):
+        steps = getattr(self, self._steps_attr)
+
+        names, estimators = zip(*steps)
+
+        name_details = [str(est) for est in estimators]
+        return _VisualBlock(
+            "serial",
```

Comment:
can we make this depend on a tag or attribute?

Reply:
just to clarify what you suggest in practice is to introduce `__init__()` to `_HeterogenousMetaEstimator` and refactor class attributes `_steps_attr` and `_steps_fitted_attr` into tags set in the init method?

Reply:
that is also an idea, but not what I mean - I meant, the value `"serial"` vs `"parallel"` passed to `_VisualBlock`, move that to be an attr or tag.